documentation/disaster-recovery.md

```diff
@@ -86,3 +93,29 @@ cf service teaching-vacancies-postgres-production
  Once the database has been successfully restored, the corrupted database instance should be deleted.
 
  `cf delete-service teaching-vacancies-<env>-old -f`
+
+## Loss of database instance
+
+In case the database instance is lost, the objectives are:
+
+- Recreate the lost postgress database instance
+- Restore data from daily backup
+
+### Recreate the lost postgress database instance
+
+In order to recreate the lost postgress database instance, use the following make recipes `terraform-app-plan` and `terraform-app-apply`:
```

Comment:
We may want to make sure the service is really gone and not in a broken state. We may have to run `cf delete-service` or even remove from terraform state

Reply:
wouldn't the terraform plan confirm that anyway?

Reply:
This may cause unexpected issues. Can we add something like:
> In case the database service is deleted or in an inconsistent state we must recreate it and repopulate it. First make sure it is full gone by running `cf services` and `cf delete-service` if necessary.
> Then recreate the lost postgress database instance using the following make recipes...